LayoutTests/svg/custom/uri-reference-handling.svg

```diff
@@ -1,4 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" onload="runTest()">
+<meta name="fuzzy" content="maxDifference=1; totalPixels=0-2157" />
```

Comment:
This hides regressions on certain platforms. E.g., the Skia rendering is currently as expected, but if it regresses we wouldn't catch that now.

Reply:
It is gradient noise - which WebKit adds (due to fingerprinting) and it leads to 1px differences and we fail some WPT due to it as well. See below, it is noise rather than regression and it is just 1px tolerance, so I don't agree that it is regression.

<img width="810" alt="image" src="https://github.com/user-attachments/assets/5e97833f-2832-4df8-9f7d-2217081ff28d" />

Reply:
Yes there are plenty of WPT gradient tests failing because of this noise. 
Where fuzziness should be added.

Reply:
I thought we only added such noise to canvas?

Reply:
We definitely do it to gradients as well.